freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/user/event/SetPasswordRequest.java

```diff
@@ -27,12 +33,17 @@ public String getPassword() {
         return password;
     }
 
+    public Optional<Instant> getExpirationInstant() {
```

Comment:
So if I understand it correctly, you allow people to add the expiration date on the set password call directly? I think we shouldn't do that. It means that anybody can "override" the policy. Let's remove it, if my understanding is correct.

Reply:
no. this is part of the internal workflow. this `SetPasswordRequest` is put on the event bus for a handler to pick up. there have been no api changes.